### ISSUE TITLE
Fabric 1.19.2 - Cant start game/server

### ISSUE THREAD

**Issue by pewblaze** [Siphalor/bouncylife#12]
Hi i cant start the game :( 
It says:
```
[13:00:46] [main/ERROR]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'tweed4_base'!
```

My log:
https://gist.github.com/pewblaze/9f447e9abfc79883fe93e1a2f4062234

**Comment by Nekkok** [comment]
Same here!

**Comment by ConqAra** [comment]
Same.

**Comment by Siphalor** [comment]
Thanks for reporting this, it should be fixed in the latest release now (1.3.1)
